### PR TITLE
feat(buyers): document guest buyer

### PR DESCRIPTION
**Pull request by gryevns**
# Description

TA-7931

Adds documentation for `buyer` prop.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own changes
- [ ] I have run `yarn lint` to make sure my changes pass all tests
- [ ] I have run `yarn test` to make sure my changes pass all linters
- [ ] I have pulled the latest changes from the upstream `main` branch
- [ ] I have tested both the react and the CDN versions on local and integration environments
- [ ] I have added the necessary labels to this PR in case a new release needs to be published after merging into `main` (e.g. `release` and `patch`)

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

File: packages/embed/README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ The options for this integration are as follows.
...
+| `buyer`                   | `null`       | An optional object for the buyer details, this cannot be used in conjunction with `buyerId` or `buyerExternalIdentifier`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```
Comment (luca-gr4vy) [inline packages/embed/README.md:68]: Same as above

**Comment by luca-gr4vy** [comment]
You can probably add those two vulnerability codes to `.trivignore` for now

**Comment by gryevns** [comment]
> You can probably add those two vulnerability codes to `.trivignore` for now

I think they should probably be fixed to meet security requirements. I tried creating a PR for axios but another one just popped up...

**Comment by luca-gr4vy** [comment]
> > You can probably add those two vulnerability codes to `.trivignore` for now
> 
> I think they should probably be fixed to meet security requirements. I tried creating a PR for axios but another one just popped up...

Usually we try to fix it by upgrading the dependency itself or any parent ones, although that might be hard. Sometimes it's enough to just resolve to the fixed version using yarn resolutions (we do that too). Not sure if this would solve it:

```
// package.json
resolutions: {
  ...
  "axios": "^1.7.4",
  "minimatch": "^4.0.8"
}
```

(then run `yarn` and commit the updated `yarn.lock`)

**Comment by gryevns** [comment]
> > > You can probably add those two vulnerability codes to `.trivignore` for now
> > 
> > 
> > I think they should probably be fixed to meet security requirements. I tried creating a PR for axios but another one just popped up...
> 
> Usually we try to fix it by upgrading the dependency itself or any parent ones, although that might be hard. Sometimes it's enough to just resolve to the fixed version using yarn resolutions (we do that too). Not sure if this would solve it:
> 
> ```
> // package.json
> resolutions: {
>   ...
>   "axios": "^1.7.4",
>   "minimatch": "^4.0.8"
> }
> ```
> 
> (then run `yarn` and commit the updated `yarn.lock`)

I had a quick look but couldn't get webpack to update. There are lots of upgrades based on the PRs open for this repo - perhaps they need to be tackled first.

File: packages/embed-react/README.md [hunk 2]
```diff
@@ -60,6 +60,7 @@ The options for this integration are as follows.
...
+| `buyer`                   | `null`      | An optional object for the buyer details, this cannot be used in conjunction with `buyerId` or `buyerExternalIdentifier`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```
[the remark this answers is not in the document]
Comment (cbetta) [reply packages/embed-react/README.md]: Not needed. That would get long

File: packages/embed/README.md (hunk 1 above)
Comment (cbetta) [reply packages/embed/README.md]: Not needed. That would get long

**Comment by luca-gr4vy** [comment]
@gryevns this should help https://github.com/gr4vy/gr4vy-embed/pull/217

**Review by luca-gr4vy — APPROVED** [review]

**Comment by gr4vy-code** [comment]
:rocket: PR was released in [`v2.29.0`](https://github.com/gr4vy/gr4vy-embed/releases/tag/v2.29.0) :rocket:

**Pull request merged**